javagdbnative/README.MD

```diff
@@ -6,22 +6,22 @@ This demo is intended to be used to demonstrate how to debug a Java application,
 1. Linux OS with GDB 10.1
 2. Visual Studio Code
 3. GraalVM Extension Pack for Java: Open Visual Studio Code, navigate to Extensions activity panel in the left-hand side Activity Bar (or use the Ctrl+Shift+X hot keys combination). Search for “GraalVM” in the search field. Find "GraalVM Extension Pack for Java", press Install. Reload will be required.
-4. [GraalVM](https://www.graalvm.org/downloads) runtime environment: Navigate to Gr activity panel in VS Code and install some of the latest GraalVM __Enterprise Edition__ versions available from the list.
+4. [GraalVM runtime environment in VS Code](https://www.graalvm.org/latest/tools/vscode/graalvm-extension/#graalvm-installation-wizard): Naviga4. [GraalVM runtime environment in VS Code](https://www.graalvm.org/latest/tools/vscode/graalvm-extension/#graalvm-installation-wizard): Navigate to Gr activity panel in VS Code and install some of the latest GraalVM __Enterprise Edition__ versions available from the list.
 5. [Native Image](https://www.graalvm.org/reference-manual/native-image/): Upon GraalVM's installation completion, the “Install Optional GraalVM Components” window pops up in the right bottom corner. Install Native Image.
```

Comment:
Update the link to https://www.graalvm.org/latest/reference-manual/native-image/.